Buildings/Fluid/HeatExchangers/ThermalWheels/Data/Characteristics/MotorEfficiency.mo

```diff
@@ -0,0 +1,36 @@
+within Buildings.Fluid.HeatExchangers.ThermalWheels.Data.Characteristics;
+record MotorEfficiency
+  "Parameters for defining motor efficiency at different wheel speed ratios"
+  extends Modelica.Icons.Record;
+
+  parameter Real uSpe[:](
+    each final min=0,
+    each final unit="1")
+    "Wheel speed ratio";
+  parameter Modelica.Units.SI.Efficiency eta[size(uSpe, 1)](
+    each final max=1)
+    "Wheel motor efficiency at a given speed ratio";
+  annotation (
+  defaultComponentPrefixes = "parameter",
+  defaultComponentName = "per",
+  Documentation(info="<html>
+<p>
+This model describes wheel speed ratio <code>uSpe</code> versus
+the motor percent full-load efficiency (see
+<a href=\"modelica://Buildings.Fluid.Movers.BaseClasses.Characteristics.motorEfficiencyCurve\">
+Buildings.Fluid.Movers.BaseClasses.Characteristics.motorEfficiencyCurve</a>).
+It is based on
```

Comment:
The development is not based on the model. However, you may instruct user to check the `Buildings.Fluid.Movers.BaseClasses.Characteristics.motorEfficiencyCurve` of how it is used.

Reply:
@JayHuLBL 
Thanks for your comment. What I was trying to convey is that this model is essentially the same as <code>Buildings.Fluid.Movers.BaseClasses.Characteristics.motorEfficiencyCurve</code>, except for the parameter definitions.
Based on your feedback, I’m now wondering if this model is actually necessary, as it might cause confusion.
Do you think I should remove it?

Reply:
@SenHuang19 We may keep it. However, I don't think the two models are same. We may not need to mention `Buildings.Fluid.Movers.BaseClasses.Characteristics.motorEfficiencyCurve` here.
Also don't need to refer it in `Buildings.Fluid.HeatExchangers.ThermalWheels.Data.Generic`.